lib/gui/models/store.js

```diff
@@ -267,10 +271,15 @@ const storeReducer = (state, action) => {
       }
 
       const selectedDevice = state.getIn([ 'selection', 'drive' ]);
+
+      /* eslint-disable lodash/prefer-lodash-method */
+
       const selectedDrive = state.get('availableDrives').find((drive) => {
         return drive.get('device') === selectedDevice;
       });
 
+      /* eslint-enable lodash/prefer-lodash-method */
+
```

Comment:
Would it be worth refactoring these two similar-looking code-blocks into a shared helper function?